tests/lib/fixtures.py

```diff
@@ -216,7 +231,9 @@ def mock_console_input(**kwargs):
                 return password_input
 
             if isinstance(user_input, Iterable):
-                return next(user_input)
+                Input_line = next(user_input)
+                # A raw newline is used to indicate deliberate empty input
+                return "" if Input_line == r"\n" else Input_line
```

Comment:
Sorry to be pedantic, but could you rename `Input_line` to `input_line` since it's a local variable?

Reply:
Never mind, I'll just do this, since it's such a tiny change.